test/tests/providers/global/harbor.all.rules.test.yml

```diff
@@ -0,0 +1,24 @@
+---
+rule_files:
+  - harbor.all.rules.yml
+tests:
+  - interval: 1m
+    input_series:
+      - series: 'harbor_up{}'
+        values: "_x5 1+0x5 0+0x5"
+    alert_rule_test:
+      - alertname: HarborDown
+        eval_time: 5m
```

Comment:
Your values say:
* from t0 to t4, no data
* from t5 to t9, "harbor_up=1"
* from t10 to t14 "harbor_down=0"

At `t5`, do you really expect an alert to fire?
I'm not sure data is added before or after the test, but either you have no data (ie probably harbor is not installed) or you have just one data point saying it's running.

This means, on clusters where harbor is not installed this alert will fire?

Reply:
Would it be a good idea to have these unit tests as separate series and values?